glxhash.c: In function ‘HashHash’:
glxhash.c:137:7: error: implicit declaration of function ‘initstate’
glxhash.c:139:10: error: implicit declaration of function ‘random’
glxhash.c:140:7: error: implicit declaration of function ‘setstate’

egllog.c: In function ‘_eglInitLogger’:
egllog.c:150:10: error: implicit declaration of function ‘strcasecmp’

--- origsrc/Mesa-8.0.5/src/egl/main/egllog.c	2012-10-24 14:03:58.000000000 -0500
+++ src/Mesa-8.0.5/src/egl/main/egllog.c	2013-10-08 19:25:44.543916600 -0500
@@ -35,6 +35,7 @@
  */
 
 
+#undef __STRICT_ANSI__
 #include <stdarg.h>
 #include <stdio.h>
 #include <stdlib.h>
--- origsrc/Mesa-8.0.5/src/glx/glxhash.c	2011-11-08 12:09:44.000000000 -0600
+++ src/Mesa-8.0.5/src/glx/glxhash.c	2013-10-08 19:25:05.334673900 -0500
@@ -70,6 +70,7 @@
  *
  */
 
+#undef __STRICT_ANSI__
 #include "glxhash.h"
 #include <X11/Xfuncproto.h>
 
